_usage/query_planning.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: Reviewing a Query Plan
+nav_order: 8
+---
+
+## Reviewing a Query Plan
+
+Babelfish version 2.1.0 includes support for functions and parameters that allow you to review estimated and actual (executed) query plans. This allows you to use the TDS port to identify and refine slow-performing queries. The functions can display an query plan for:
+
+- `SELECT`, `INSERT`, `UPDATE`, and `DELETE` statements
+- Nested procedures, multiple statements in a batch
+- Variable declarations (for example, `DECLARE @t TABLE (a int, b int);`)
+
+The functions *do not* support query plan retrieval from functions, control flows, and cursors.
+
+You can use a SET statement on the TDS port to turn on/off the following functions: 
+
+- SET BABELFISH_STATISTICS PROFILE {ON|OFF} to display the query plan used to execute a statement. The command implements the behavior of the [PostgreSQL EXPLAIN ANALYZE statement] (https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-ANALYZE).
+    
+- SET BABELFISH_SHOWPLAN_ALL {ON|OFF} to display estimated execution plans for a statement without performing the command. The command implements the behavior of the [PostgreSQL EXPLAIN statement](https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-BASICS).
+
+For example, the following command sequence turns on query planning, returns the query plan for the SELECT statement without executing the command, and then turns off query planning:
```

Comment:
remove 'command'.  while you can argue that 'SET" is a command (it's a statement. really), SELECT certainly is not a command.

Reply:
Now reads: The following example turns on query planning, returns the query plan for the SELECT statement without executing the command, and then turns off query planning:

Reply:
Now reads: The following example turns on query planning, returns the query plan for the SELECT statement without executing the command, and then turns off query planning: